frame/auction.ts

```diff
@@ -39,3 +62,48 @@ function randomToken() {
     )
     .join("");
 }
+
+async function fetchAndValidateTrustedScoringSignals(
+  baseUrl: string,
+  hostname: string,
+  renderingUrls: ReadonlySet<string>
+) {
+  const url = new URL(baseUrl);
+  url.searchParams.append("hostname", hostname);
+  url.searchParams.append(
+    "keys",
+    [...renderingUrls].map(encodeURIComponent).join(",")
+  );
+  const response = await tryFetchJson(url.href);
```

Comment:
We should be verifying that the response has `X-ALLOW: fledge`

Reply:
Doing this in #62.